apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -56,16 +90,45 @@ class ManageStudentActionsCell extends Component {
             <Button onClick={this.onCancel} color={Button.ButtonColor.blue} text="Cancel" />
           </div>
         }
+        <BaseDialog
+          useUpdatedStyles
+          uncloseable
+          isOpen={this.state.deleting}
+          style={{paddingLeft: 20, paddingRight: 20, paddingBottom: 20}}
+        >
+          <h2 style={styles.heading}>Remove student</h2>
+          <div>Are you sure you want to remove this student?</div>
```

Comment:
will need i18n presumably

I might even have this dialog as it's own component (even if you just have it as a separate component elsewhere in this same file)